lib/medtronic/processData.js

```diff
@@ -472,7 +472,7 @@ function buildBGRecords(records) {
 function buildCGMRecords(events) {
 
   var postrecords = [];
-  var start = null;
+  var start = events[events.length-1].jsDate;
```

Comment:
For CGM events, yes. And in this specific instance, putting in a check wouldn't really help much, as `start` will then either be `null` or `undefined`.